docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -237,7 +241,15 @@ First - find the correct **GetCapabilities URL**. Best way to achieve it is by l
 
 ``` xml
 <mc:links scheme="WMTS" name="[desired_layer_identifier]" description="">
-  '<RASTER-RASTER-SERVING-SERVICE_URL>/wmts/1.0.0/WMTSCapabilities.xml'
+  '<RASTER-SERVING-SERVICE_URL>/wmts/1.0.0/WMTSCapabilities.xml'
+</mc:links>
+```
+
+**OR** you can use wmts kvp capabilities
+
+``` xml
```

Comment:
Note that OL / cesium examples supposed to work with the first capabilities, for the second capabilities or mention in the example that it won't work or supply a valid example for kvp capabilities

Reply:
the point is that the second capabilities (KVP) works the exactly same way as the first one (RESTfull) no changes need, i did check the scenario with Alex's support to validate it.

Reply:
see https://cesium.com/learn/cesiumjs/ref-doc/WebMapTileServiceImageryProvider.html

Reply:
OL doesn't work the same

Reply:
just checked with OpenLayers, works exactly the same way. :)

Reply:
👍